src/index.html

```diff
@@ -22,6 +22,342 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#moyo-logo"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/093d4676-a745-433f-847c-4a8b79d4a6d4)